### ISSUE TITLE
Cannot call a class as a function

### ISSUE THREAD

**Issue by elchicofrommo** [73R3WY/react-native-svg-animations#29]
Hi team, 

I'm having a weird error with AnimiatedSVGPath at line 55. It took some effort but seems like my set up is complaining that I'm trying to treat svgPathProperties as a function when it's a class. I assume it has to do with the way my env is set up. I'm using Expo Managed with Typescript (not bare) Below is my package.json typescript expo and babel config. Anyone able to help me figure what about my set up is causing the error? Or maybe do a pull request to use 
`new svgPathProperties(d)` instead. 

Thanks

package.json
```
{
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web",
    "eject": "expo eject"
  },
  "dependencies": {
    "@fortawesome/fontawesome-svg-core": "^1.2.28",
    "@fortawesome/free-solid-svg-icons": "^5.13.0",
    "@fortawesome/react-native-fontawesome": "^0.2.5",
    "@react-native-community/masked-view": "0.1.6",
    "@react-navigation/native": "^5.4.0",
    "@react-navigation/stack": "^5.3.7",
    "expo": "~37.0.3",
    "react": "~16.9.0",
    "react-dom": "~16.9.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
    "react-native-flip-card": "^3.5.6",
    "react-native-gesture-handler": "~1.6.0",
    "react-native-safe-area-context": "0.7.3",
    "react-native-screens": "~2.2.0",
    "react-native-snap-carousel": "^3.9.0",
    "react-native-svg": "11.0.1",
    "react-native-svg-animations": "^0.2.0",
    "react-native-svg-transformer": "^0.14.3",
    "react-native-swiper": "^1.6.0",
    "react-native-web": "~0.11.7",
    "svg-path-properties": "^1.0.4"
  },
  "devDependencies": {
    "@babel/core": "^7.8.6",
    "@types/react": "~16.9.23",
    "@types/react-native": "~0.61.17",
    "@types/react-native-snap-carousel": "^3.8.1",
    "babel-preset-expo": "~8.1.0",
    "typescript": "~3.8.3"
  },
  "private": true
}
```

typescrpt
```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "jsx": "react-native",
    "lib": ["dom", "esnext"],
    "moduleResolution": "node",
    "noEmit": true,
    "skipLibCheck": true,
    "resolveJsonModule": true,
    "strict": true
  }
}
```

babel
```
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
  };
};
```

expo config
```
{
  "expo": {
    "name": "flashcards",
    "slug": "flashcards",
    "platforms": [
      "ios",
      "android",
      "web"
    ],
    "version": "1.0.0",
    "orientation": "portrait",
    "icon": "./assets/icon.png",
    "splash": {
      "image": "./assets/splash.png",
      "resizeMode": "contain",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": [
      "**/*"
    ],
    "ios": {
      "supportsTablet": true
    },
    "description": "AWS training flashcard"
  }
}

```

**Comment by 73R3WY** [comment]
Hi @elchicofrommo , thank you for the feedback, there is an outstanding pull request already for this issue, it is #27 . I will be merging it today since it should fix the issue apparently.

**Comment by 73R3WY** [comment]
Closing this, merged #27 and updated npm package to `0.2.1`.

**Comment by elchicofrommo** [comment]
awesome, thanks.
